cfpb/cfgov-refresh
# Ask CFPB: Pick up legacy English tag searches

We don't offer tag links on English Answer pages (as we do on legacy
Spanish Answer pages), but it turns out some existing cfgov pages were
linking to English question collections via tag-based facet queries. 
In order to rescue those links, we need to tweak ask_cfpb's search redirect view, its routable
tag pages and its English search results template.

## Additions
- This post-migration PR depends on the existence of an English TagResultsPage (one of those  RoutablePageMixin celebrities) to handle tag-based search requests.

## Testing
- If you have a production db download pulled on 6/27 or later, the results page will be created. You'll just need to [open the page](http://localhost:8000/admin/pages/10138/edit/) and publish it.
- Alternatively, you can create it yourself in your local admin. 
    - Navigate to the Ask CFPB landing page, and add a child page using the TagResultsPage type. 
    - Give it any title you want, but edit **Configuration > slug** to be "search-by-tag".
    - Publish the page.
 - Then enter a legacy English tag-facet page URL, and it should redirect  
to the new `/search-by-tag/` URL.   

Example:  
The old URL http://localhost:8000/askcfpb/search/?selected_facets=tag_exact:middle+school   
should redirect to http://localhost:8000/ask-cfpb/search-by-tag/middle_school/
